content/_layouts/developerguide.html.haml

```diff
@@ -0,0 +1,5 @@
+---
+layout: developer
```

Comment:
I created a layout for each kind of page to prepare for future presentation differences (as today already in `chapter`), and to be able to see at a glance from the source what a document is used for.

`developer` is just the "abstract superclass" of the three developer.+ layouts.